core/src/avm1/activation.rs

```diff
@@ -988,16 +988,19 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
     fn action_enumerate(&mut self) -> Result<FrameControl<'gc>, Error<'gc>> {
         let name_value = self.context.avm1.pop();
         let name = name_value.coerce_to_string(self)?;
-        self.context.avm1.push(Value::Null); // Sentinel that indicates end of enumeration
         let object: Value<'gc> = self.get_variable(name)?.into();
+        self.context.avm1.push(Value::Undefined); // Sentinel that indicates end of enumeration
 
         match object {
             Value::Object(ob) => {
                 for k in ob.get_keys(self).into_iter().rev() {
                     self.context.avm1.push(k.into());
                 }
             }
-            _ => avm_error!(self, "Cannot enumerate properties of {}", name),
+            _ => {
+                //TODO: when ErrorReportingEnable=1 in mm.cfg the following is traced on error: "Warning {} is not defined"
```

Comment:
As far as I know, `mm.cfg` applies only for Flash Player Debug Projector, and has nothing to do in the "release" Flash Player. So we probably should not try to trace a warning here.